site24x7/provider_test.go

```diff
@@ -1,35 +1,27 @@
 package site24x7
 
-import (
-	"os"
-	"testing"
-
-	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
-	"github.com/hashicorp/terraform-plugin-sdk/terraform"
-)
-
-var testAccProviders map[string]terraform.ResourceProvider
-var testAccProvider *schema.Provider
-
-func init() {
-	testAccProvider = Provider().(*schema.Provider)
-	testAccProviders = map[string]terraform.ResourceProvider{
-		"site24x7": testAccProvider,
-	}
-}
-
-func TestProvider(t *testing.T) {
-	if err := Provider().(*schema.Provider).InternalValidate(); err != nil {
-		t.Fatalf("err: %s", err)
-	}
-}
-
-func TestProvider_impl(t *testing.T) {
-	var _ terraform.ResourceProvider = Provider()
-}
-
-func testAccPreCheck(t *testing.T) {
-	if v := os.Getenv("ZOHO_AUTHTOKEN"); v == "" {
-		t.Fatal("ZOHO_AUTHTOKEN must be set for acceptance tests")
-	}
-}
+//var testAccProviders map[string]terraform.ResourceProvider
+//var testAccProvider *schema.Provider
+//
+//func init() {
+//	testAccProvider = Provider().(*schema.Provider)
+//	testAccProviders = map[string]terraform.ResourceProvider{
+//		"site24x7": testAccProvider,
+//	}
+//}
+//
+//func TestProvider(t *testing.T) {
+//	if err := Provider().(*schema.Provider).InternalValidate(); err != nil {
+//		t.Fatalf("err: %s", err)
+//	}
+//}
+//
+//func TestProvider_impl(t *testing.T) {
+//	var _ terraform.ResourceProvider = Provider()
+//}
+//
+//func testAccPreCheck(t *testing.T) {
+//	if v := os.Getenv("ZOHO_AUTHTOKEN"); v == "" {
+//		t.Fatal("ZOHO_AUTHTOKEN must be set for acceptance tests")
+//	}
+//}
```

Comment:
Any reason you commented this?

Reply:
This is not needed, I will comment it in again

Reply:
Then just leave it there. We will address that in another PR as is does not have anything to do with the feature you're implementing in this PR ;).